### PR TITLE
Breadcrumb component

### PR DESCRIPTION
**Pull request by benilovj**
### Context

Breadcrumb markup is repetitive and hard to parse at a glance. We want to abstract it away to clean up view templates.

### Changes proposed in this pull request

- Introduce a breadcrumb component and a view helper
- Apply it in the first view

### Guidance to review

Usage example:

```ruby
<%- content_for :before_content do %>
  <% breadcrumb([
    { t('page_titles.responsible_body_home') => responsible_body_home_path },
    title,
  ]) %>
<% end %>
```

Is the syntactic sugar sensible?

### CODE REVIEW

File: app/helpers/view_helper.rb [hunk 1]
```diff
@@ -30,6 +30,10 @@ def link_to_devices_guidance_subpage(body, slug, html_options = {})
     govuk_link_to body, devices_guidance_subpage_path(subpage_slug: slug), html_options
   end
 
+  def breadcrumb(items)
```
Comment (aldavidson) [inline app/helpers/view_helper.rb:33]: just one suggestion about the name of this: the singular 'breadcrumb' to me implies that this is for one component of the breadcrumbs/breadcrumb trail. Can I suggest calling it 'breadcrumbs' or 'breadcrumb_trail' instead?

**Review by aldavidson — APPROVED** [review]
Looks great, and pretty much exactly what I was thinking. Just one suggestion about the name of the helper method (not a blocker tho)

File: app/helpers/view_helper.rb (hunk 1 above)
Comment (benilovj) [reply app/helpers/view_helper.rb]: yep, good suggestion, fixed in 8d5acc7
